packages/app/obojobo-repository/shared/components/dashboard.jsx

```diff
@@ -12,6 +12,7 @@ const Button = require('./button')
 const MultiButton = require('./multi-button')
 const Search = require('./search')
 const ReactModal = require('react-modal')
+const APIUtil = require('obojobo-document-engine/src/scripts/viewer/util/api-util')
```

Comment:
I'd prefer not to use the viewer's api-util here, it has a lot more then we need in it